update vpce id info (#9205)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/secure/private-connectivity/aws/snowflake.md

## Configuring network policies
If your organization uses [Snowflake Network Policies](https://docs.snowflake.com/en/user-guide/network-policies) to restrict access to your Snowflake account, you need to add a network rule for <Constant name="dbt" />. 

You need a VPCE ID to create a network policy in Snowflake:
1. In <Constant name="dbt_platform" />, go to **Account settings → Integrations → Private endpoints** 

> 2. Open your endpoint to find its VPCE ID on the endpoint details page. 

Reviewer: ```suggestion
2. Open your endpoint and locate its **VPCE ID** field on the endpoint details page. 
```

## Configuring network policies
If your organization uses [Snowflake Network Policies](https://docs.snowflake.com/en/user-guide/network-policies) to restrict access to your Snowflake account, you need to add a network rule for <Constant name="dbt" />. 

> You need a VPCE ID to create a network policy in Snowflake. In <Constant name="dbt_platform" />, go to **Account settings → Integrations → Private endpoints** and open your endpoint to find its VPCE ID on the endpoint details page. If you configured PrivateLink through [Support-led setup](#support-led-setup), or **Private endpoints** is not available in your account settings, contact [<Constant name="dbt" /> Support](mailto:support@getdbt.com) to obtain the VPCE ID. If you're creating an endpoint for Internal Stage, the VPCE ID is different from the VPCE ID for the main service endpoint.

Author: ```suggestion
You need a VPCE ID to create a network policy in Snowflake:
1. In <Constant name="dbt_platform" />, go to **Account settings → Integrations → Private endpoints** 
2. Open your endpoint to find its VPCE ID on the endpoint details page. 
3. If you configured PrivateLink through [Support-led setup](#support-led-setup), or **Private endpoints** is not available in your account settings, contact [<Constant name="dbt" /> Support](mailto:support@getdbt.com) to obtain the VPCE ID. 
4. If you're creating an endpoint for Internal Stage, the VPCE ID is different from the VPCE ID for the main service endpoint.
```

4. Give the rule a name.
5. Select a database and schema where the rule will be stored. These selections are for permission settings and organizational purposes; they do not affect the rule itself.
6. Set the type to `AWS VPCE ID` and the mode to `Ingress`.

> 7. Type the VPCE ID from the endpoint details page in <Constant name="dbt_platform" /> (or from <Constant name="dbt" /> Support if you used Support-led setup) into the identifier box and press **Enter**.

Reviewer: Suggest removing "press enter" unless that's what it actually says on the user screen. It's "Enter" on Windows keyboards, "Return" on Mac/Linux, and just a symbol on others
```suggestion
7. Enter the VPCE ID from the endpoint details page in <Constant name="dbt_platform" /> (or from <Constant name="dbt" /> Support if you used Support-led setup) into the identifier box.
```

CREATE NETWORK RULE allow_dbt_cloud_access
  MODE = INGRESS
  TYPE = AWSVPCEID

>   VALUE_LIST = ('<VPCE_ID>'); -- Replace '<VPCE_ID>' with the VPCE ID from your endpoint details page or from Support

Author: ```suggestion
  VALUE_LIST = ('<VPCE_ID>'); -- Replace '<VPCE_ID>' with the VPCE ID the actual value
```
